src/test/java/com/pro/sky/employeesaccounting/service/DepartmentServiceTest.java

```diff
@@ -0,0 +1,72 @@
+package com.pro.sky.employeesaccounting.service;
+
+import Model.Employee;
+import exception.EmployeeNotFoundException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import service.DepartmentService;
+import service.DepartmentServiceImpl;
+import service.EmployeeService;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+import static org.skyscreamer.jsonassert.JSONAssert.assertEquals;
```

Comment:
Не тот импорт, assertEquals нужен из junit
```suggestion
import static org.junit.jupiter.api.Assertions.assertEquals;
```